dir.targets

```diff
@@ -43,6 +43,7 @@
     <!-- if building desktop facade, we don't bin place the refs -->
     <BinPlaceRef Condition="'$(BinPlaceRef)' == '' And '$(BuildingDesktopFacade)' != 'true' And ('$(IsReferenceAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRef>
     <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' And ('$(IsRuntimeAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRuntime>
+    <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' And ('$(IsNETCoreApp)' == 'true' AND '$(IsTestProject)' == 'true' AND '$(BinPlaceNETCoreAppTests)' == 'true')">true</BinPlaceRuntime>
```

Comment:
-p:ArchiveTests=true does what I wanted except filtering non-netcoreapp tests. I tried the following patch to implement filtering, but it didn't work:
```
diff --git a/Tools-Override/publishtest.targets b/Tools-Override/publishtest.targets
index a813902..8ef196b 100644
--- a/Tools-Override/publishtest.targets
+++ b/Tools-Override/publishtest.targets
@@ -9,6 +9,7 @@

   <PropertyGroup>
     <SerializeProjects Condition="'$(TestWithLocalLibraries)'=='true'">true</SerializeProjects>
+    <ArchiveTests Condition="'$(ArchiveNETCoreAppTests)' == 'true' AND '$(BuildingNETCoreAppVertical)' != 'true'">false</ArchiveTests>
   </PropertyGroup>

   <!--
```
Where should I change in order to filter non netcoreapp tests during archiving tests? (I do not care about over building, but it would be better if you prevent over building).